centos2ol.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
-# Copyright (c) 2020 Oracle and/or its affiliates.
+# Copyright (c) 2020-2021 Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 #
-# Script to switch CentOS (or other similar distribution) to the
-# Oracle Linux yum repository.
-#
+# Script to switch CentOS (or other similar distribution) to the Oracle Linux yum repository.
+# with better diagnistics output formatting and restore_repos function converted to a dummy. 
+# 1.00  oracle  2020/12/28  Oracle initial implementation
+# 2.00  bezroun 2021/01/07  Some cosmetic changes and better diagnistic 
+
 
 set -e
 unset CDPATH
+declare VERSION='2.00'
```

Comment:
We should probably stick to semantic versioning (X.Y.Z) which means this would be `1.1.0` not `2.0`.